Symposium/BasicModal.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { Modal, StyleSheet, View, KeyboardAvoidingView, Keyboard, FlatList, TouchableOpacity, TouchableWithoutFeedback, ScrollView } from 'react-native';
+import { Stack, TextInput, IconButton, HStack } from "@react-native-material/core";
+import { Button as Rbutton } from "@react-native-material/core";
+
+import Icon from "@expo/vector-icons/MaterialCommunityIcons";
+import { Card, Text, Button } from 'react-native-paper';
+const BasicModal = (props) => {
+    const topicList = ["Entertainment", "Politics", "IT", "Business"]
+    const [modalVisible, setModalVisible] = useState(false);
+    return (
+        <View style={styles.centeredView}>
+
+            <Modal
+                animationType="slide"
+                transparent={true}
+                visible={modalVisible}
+                onBackdropPress={() => setModalVisible(false)}
+                onRequestClose={() => {
+                    setModalVisible(!modalVisible)
+                }}
+            >
+                <View
+                    style={styles.modalContainer}>
+                    <KeyboardAvoidingView
+                        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+                        style={{ flex: 1 }}
+                    >
+                        <View style={styles.modalView}>
+                            <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+                                <Text style={styles.modalTitle}>Have your say!</Text>
+                            </TouchableWithoutFeedback>
+                            <Stack fill spacing={8} style={{ width: "100%", marginTop: 16 }}>
+                                <ScrollView style={{ flex: 1 }}>
+                                    <TextInput
+                                        style={{ width: "100%" }}
+                                        variant='outlined'
+                                        label='Enter your subject...'
```

Comment:
Please use `placeholder` instead of `label` so that text doesn't overlap.
```suggestion
                                        placeholder='Enter your subject...'
```
![image](https://user-images.githubusercontent.com/25761885/233370932-6d6a4ed4-a0bc-4ad2-b54a-0c7339a66ecd.png)

Reply:
what is the issue you meant? adding margin seems to be the trick to me

Reply:
<img height="440" src="https://user-images.githubusercontent.com/55075292/233562657-8ebccaeb-735a-4170-ba0a-f1bf1291a1e9.jpeg"/>

Reply:
When you write text and deselect the text box, both the label and the entered text overlap.

Reply:
resolved in the latest commit! thanks Mark!